lib/metadataTypes/Automation.js

```diff
@@ -147,12 +139,18 @@ class Automation extends MetadataType {
                             'CreatedBy',
                             'CreatedDate',
                         ],
-                        { options: options }
-                    );
-                })
+                        {
+                            filter: {
+                                leftOperand: 'ProgramID',
+                                operator: 'equals',
+                                rightOperand: a.ObjectID,
+                            },
+                        }
+                    )
+                )
             )
         ).forEach((item) => {
-            details.push(...item);
+            details.push(...item.Results);
```

Comment:
my preferred structure here would be array.map(item=>item.Results).flat() - technically its not wrong, but usually map.flat would be preferred over foreach

Reply:
i hate map - and then flat on top heeeeeeeeelp
:D